### ISSUE TITLE
Non-ascii symbols are not displayed in notifications

### ISSUE THREAD

**Issue by tukaef** [datalust/seq-tickets#652]
Hi,

I just tried the new Seq build `4.2.470`. 

It seems like notification apps are breaking some unicode symbols.

```
Log.Logger = new LoggerConfiguration()
                         .WriteTo.Seq("http://localhost:5341")
                         .CreateLogger();

Log.Information("New message: Привет ǐ ȏ ȱ");
Log.CloseAndFlush();
```

At the target site I've got:
`New message: ������ i ? ?`

I've tested this on `Seq.App.EmailPlus v2.0.78` and `Seq.App.Slack v0.2.23` apps. The same versions worked fine on build `4.1.17`.

Diagnostics log:
```
====================
1. Executable
--------------------
Seq Version         : 4.2.470
Bitness             : 64
Framework Version   : 4.0.30319.42000

====================
2. Operating System
--------------------
OS Caption          : Microsoft Windows 10 Pro
OS Version          : 10.0.15063
OS Bitness          : 64
```

**Comment by nblumhardt** [comment]
Thanks for the report, Alexander.

Sadly, we had this covered, but a later refactoring introduce a regression that wasn't picked up. We'll release a patch with the fix as soon as possible.

**Comment by nblumhardt** [comment]
Confirmed fixed in build 4.2.476

![image](https://user-images.githubusercontent.com/342712/35070833-64233c40-fc2a-11e7-8fef-3a1fb4edf3ba.png)

**Comment by tukaef** [comment]
Perfect, many thanks!
